module-4/backend/week_two.md

```diff
@@ -5,18 +5,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+
+  - in ES6, you can declare functions with the fat arrow notation (() => {})
+  - ES6 introduces require('..') and classes
+  - ES5 requires the use of semi-colons in some instances
+
 2. What's the difference between asynchronous and synchronous JavaScript? 
+
+  Asynchronous JavaScript pops promises and API calls off the execution stack and adds them to an external queue. When the promise's conditions are met, the code block is re-added to the stack. Synchronous JavaScript, alternately, is put on the execution stack one by one in a predictable order.
```

Comment:
Perfect! Make sure you're clear that JS is synchronous by nature; but we have some functions that behave asynchronously.